hs_geo_raster_resource/raster_meta_extract.py

```diff
@@ -137,36 +145,44 @@ def get_wgs84_coverage_info(raster_dataset):
-    original_cs.ImportFromWkt(raster_dataset.GetProjection())
-
-    # create wgs84 geographic coordinate system
-    wgs84_cs = osr.SpatialReference()
-    wgs84_cs.ImportFromEPSG(4326)
-
-    # get original bounding box info
-    original_coverage_info = get_original_coverage_info(raster_dataset)
-    original_northlimit = original_coverage_info['northlimit']
-    original_southlimit = original_coverage_info['southlimit']
-    original_westlimit = original_coverage_info['westlimit']
-    original_eastlimit = original_coverage_info['eastlimit']
-
-    # create transform object
-    transform = osr.CoordinateTransformation(original_cs, wgs84_cs)
-    if transform.this is not None:
-        # transform original bounding box to wgs84 bounding box
-        wgs84_westlimit,wgs84_northlimit = transform.TransformPoint(original_westlimit, original_northlimit)[:2]
-        wgs84_eastlimit,wgs84_southlimit = transform.TransformPoint(original_eastlimit, original_southlimit)[:2]
-
-        wgs84_coverage_info = OrderedDict([
-            ('northlimit', wgs84_northlimit),
-            ('southlimit', wgs84_southlimit),
-            ('eastlimit', wgs84_eastlimit),
-            ('westlimit', wgs84_westlimit),
-            ('units','Decimal degrees'),
-            ('projection', 'WGS 84 EPSG:4326')
-        ])
-        return wgs84_coverage_info
+    try:
+        proj = raster_dataset.GetProjection()
+    except:
+        proj = None
+
+    if proj:
+        original_cs = osr.SpatialReference()
+        original_cs.ImportFromWkt(proj)
+
+        # create wgs84 geographic coordinate system
+        wgs84_cs = osr.SpatialReference()
+        wgs84_cs.ImportFromEPSG(4326)
+
+        # get original bounding box info
+        original_coverage_info = get_original_coverage_info(raster_dataset)
+        original_northlimit = original_coverage_info['northlimit']
+        original_southlimit = original_coverage_info['southlimit']
+        original_westlimit = original_coverage_info['westlimit']
+        original_eastlimit = original_coverage_info['eastlimit']
+
+        # create transform object
+        transform = osr.CoordinateTransformation(original_cs, wgs84_cs)
+        if transform.this is not None:
+            # transform original bounding box to wgs84 bounding box
+            wgs84_westlimit,wgs84_northlimit = transform.TransformPoint(original_westlimit, original_northlimit)[:2]
+            wgs84_eastlimit,wgs84_southlimit = transform.TransformPoint(original_eastlimit, original_southlimit)[:2]
+
+            wgs84_coverage_info = OrderedDict([
+                ('northlimit', wgs84_northlimit),
+                ('southlimit', wgs84_southlimit),
+                ('eastlimit', wgs84_eastlimit),
+                ('westlimit', wgs84_westlimit),
+                ('units','Decimal degrees'),
+                ('projection', 'WGS 84 EPSG:4326')
+            ])
+            return wgs84_coverage_info
+        else:
+            return None
```

Comment:
It is not. That is for fixing another raster resource type specific issue. I made the fix in this branch for testing purposes, then created an issue branch to address that specific raster resource issue, and already had that specific pull request merged into develop. Since then, I have not rebased with develop branch, which probably is why it shows here as difference. I did merged develop into this branch yesterday afternoon, but did not push the changes into this branch due to user verification no longer working after merging.